libs/datasets/dataset_utils.py

```diff
@@ -142,6 +142,18 @@ def check_index_values_are_unique(data, index=None, duplicates_as_error=True):
     return None
 
 
+def decorate_new_cases(multiregion_timeseries):
```

Comment:
nit: `decorate` feels a bit misleading to me only because of the presence of python decorators.  My first thought is that this function would be a decorator

Reply:
Ah, old habit :-) Happy to call it something more obvious!